metricbeat/docs/metricbeat-options.asciidoc

```diff
@@ -1,30 +1,85 @@
 [[configuration-metricbeat]]
 == Specify which modules to run
 
-To enable specific modules and metricsets, you add entries to the `modules` list in of the +{beatname_lc}.yml+ config file. Each entry in the list begins with a dash (-) and is followed by settings for that <<metricbeat-modules,module>>.
+Metricbeat provides a couple different ways to enable modules and metricsets:
 
-//REVIWERS: Using "list" here instead of "array" to match the terminology we use in the Platform Ref to describe the config file
+* <<enable-modules-d-configs>>
+* <<enable-modules-config-file>>
+
+[float]
+[[enable-modules-d-configs]]
+=== Enable module configs in the `modules.d` directory
+
+The `modules.d` directory contains default configurations for all the modules
+available in Metricbeat. You can enable or disable specific module
+configurations  under `modules.d` by running the <<modules-command,`modules
+enable` or `modules disable`>> commands.
+
+For example, to enable the `apache` and `mysql` configs in the `modules.d`
+directory, you use:
+
+[source,shell]
+----
+./metricbeat modules enable apache mysql
+----
+
+Then when you run Metricbeat, it loads the corresponding module configurations
+specified in the `modules.d` directory (for example, `modules.d/apache.yml` and
+`modules.d/mysql.yml`).
+
+To see a list of enabled and disabled modules, run:
+
+[source,shell]
+----
+./metricbeat modules list
+----
+
+You can change the default module configurations by modifying the `.yml` files
+in the `modules.d` directory. 
 
 The following example shows a basic configuration for the Apache module:
 
 [source,yaml]
 ----
-metricbeat:
-  modules:
-    - module: apache
-      metricsets: ["status"]
-      hosts: ["http://127.0.0.1/"]
-      period: 10s
-      enabled: true
-      fields:
-        dc: west
-      tags: ["tag"]
-      processors:
-        ....
+- module: apache
+  metricsets: ["status"]
+  hosts: ["http://127.0.0.1/"]
+  period: 10s
+  enabled: true
+  fields:
+    dc: west
+  tags: ["tag"]
+  processors:
+  ....
 ----
 
+// REVIEWERS: We aren't very consistent about when to show enabled: true. I'm
+// assuming this option is available for all modules, but wondering if we should
+// show it in the config.
```

Comment:
We had some discussion about `enabled: true`, the idea is that having that it's the default and we provide an alternative to manage enabled/disabled status (renaming the yml file), it looks like a good idea to hide that so we don't provide to ways of doing the same thing

Reply:
Ok, I'll remove it from the examples then.